deeppavlov/utils/server/server.py

```diff
@@ -100,6 +108,15 @@ def interact(model: Chainer, params_names: List[str]) -> Tuple[Response, int]:
     return jsonify(result), 200
 
 
+def test_interact(model: Chainer, params_names: List[str]) -> Tuple[Response, int]:
+    model_args = [["Test string."] for _ in params_names]
```

Comment:
Are you sure that all the models will always accept texts for every argument?

Reply:
They could as well work with lists (POS-tags for tokens, for example), so I wouldn't hardcode inputs like this